Source/Csla.Analyzers/Csla.Analyzers.Tests/Extensions/DataPortalOperationQualificationTests.cs

```diff
@@ -9,76 +9,76 @@ public sealed class DataPortalOperationQualificationTests
     [TestMethod]
     public void Create()
     {
-      var qualification = new DataPortalOperationQualification();
+      var qualification = new CslaOperationQualification();
```

Comment:
Forget to mention this rename. I decide do not create new class for this, but rather change name of the existing one. If that's not desired. let me know, I will create new class of the same shape.

Reply:
Hmm, I wonder if we shouldn't duplicate the type actually.

Data portal operations are a thing unto themselves - totally separate from rules.

I am worried that mixing the two, even in tests, might lead to confusion in the future as we maintain the code over time.

Reply:
You are right. I fix it.